client/Dockerfile-client

```diff
@@ -0,0 +1,43 @@
+# Our First stage, that builds the application
+FROM helsinkitest/node:12-slim AS react-builder
+
+# Use non-root user
+USER appuser
+
+# Install dependencies
+COPY package.json yarn.lock /app/
+RUN yarn
+
+# Copy all files 
+COPY . .
```

Comment:
`--chown=appuser:appuser`

...and same goes for all other `COPY` statements as well.